atomix/cluster/src/main/java/io/atomix/raft/impl/RaftContext.java

```diff
@@ -617,12 +609,6 @@ public void close() {
 
     // close thread contexts
     threadContext.close();
-    loadContext.close();
-
-    // Only close the thread context factory if indicated.
-    if (closeOnStop) {
-      threadContextFactory.close();
```

Comment:
Why it was necessary to close the factory before? :thinking:

Reply:
Depends on who is the owner of the threadFactory. If it was provided from RaftPartition, then it will be closed there. If it was created by DefaultRaftServer, then it is closed here. Anyway, threadContextFactory was not used. So this would not change anything.